lib/Tumblr/StreamBuilder/FencepostRanking/CacheFencepostProvider.php

```diff
@@ -95,7 +101,8 @@ public function set_latest_timestamp(string $fence_id, int $timestamp_ms)
         $this->cache->set(
             CacheProvider::OBJECT_TYPE_FENCEPOST_ID,
             self::cachekey_fence($fence_id),
-            $timestamp_ms
+            $timestamp_ms,
+            $this->ttl_seconds
```

Comment:
i think `set_fencepost_epoch()` needs to be updated as well

Reply:
also `set_fencepost`